corehq/messaging/scheduling/forms.py

```diff
@@ -2816,6 +2847,18 @@ class ConditionalAlertScheduleForm(ScheduleForm):
         required=False,
     )
 
+    sms_stale_after = IntegerField(
+        label=ugettext_lazy("Make SMS stale after (days)"),
+        required=False,
+        min_value=MIN_SMS_STALE_INTERVAL,
+        max_value=MAX_SMS_STALE_INTERVAL,
+        help_text=ugettext_lazy("SMS is marked as stale in 7*24 hours by default.\
+            You can set minimum value to {min} hours and maximum value to {max} hours").format(
```

Comment:
Nitpick: since the value is now in days, the help should also talk about days, not hours.

Reply:
Oops, I was reading commits out of order, the final value is in days now. So this is good, but the label above should say hours.